src/routes/+page.svelte

```diff
@@ -12,6 +12,8 @@
 	import { HeadingNode, QuoteNode, registerRichText } from '@lexical/rich-text';
 	import { mergeRegister } from '@lexical/utils';
 	import { createEditor } from 'lexical';
+	import { registerMarkdownShortcuts, DEFAULT_TRANSFORMERS } from '@lexical/markdown';
+	import { $isRootTextContentEmptyCurry as isRootTextContentEmptyCurry } from '@lexical/text';
```

Comment:
this is the source of the error

```
09:45:29 [vite] Error when evaluating SSR module /src/routes/+page.svelte: failed to import "/node_modules/.pnpm/@lexical+text@0.14.2_lexical@0.14.2/node_modules/@lexical/text/LexicalText.esm.js"
|- SyntaxError: Unexpected token ':'
    at new AsyncFunction (<anonymous>)
    at instantiateModule (file:///home/etrepum/lexical-esm-sveltekit-vanilla-js/node_modules/.pnpm/vite@5.2.6/node_modules/vite/dist/node/chunks/dep-BBHrJRja.js:55052:28)

SyntaxError: Unexpected token ':'
    at new AsyncFunction (<anonymous>)
    at instantiateModule (file:///home/etrepum/lexical-esm-sveltekit-vanilla-js/node_modules/.pnpm/vite@5.2.6/node_modules/vite/dist/node/chunks/dep-BBHrJRja.js:55052:28)
```

Reply:
This should be fixed after the release of https://github.com/facebook/lexical/pull/5737 - but there are other reasons why the code in the PR won't work (DEFAULT_TRANSFORMERS isn't exported, and registerMarkdownShortcuts requires that all of the nodes it uses be implemented where this one is more minimal than its default set of transforms)

Reply:
Yeah i opened this PR on accident 😅 i ended up sharing a different one on Discord and forgot to close this